playwright/helper.py

```diff
@@ -243,3 +243,14 @@ class RouteHandlerEntry:
     def __init__(self, matcher: URLMatcher, handler: RouteHandler):
         self.matcher = matcher
         self.handler = handler
+
+def raise_not_installed_error(message: str) -> None:
+    raise Exception(f"""
```

Comment:
nit: Probably I would prefer to create a new Error class which accepts in the constructor the browser name.

Reply:
Ah might be irrelevant since it does not only accept browser names, idk.